distributionviewer/core/static/css/chart-detail.styl

```diff
@@ -3,24 +3,13 @@
 .chart-detail {
   width: 100%;
 
-  .chart {
-    height: 60vh;
-    margin: $med-padding auto $lrg-padding;
-    width: 100%;
+  .chart-list-name {
```

Comment:
I would `font-weight: 100` or 200 this so that it doesn't seem out of place with its degree of boldness.